codex/slots/proofs/backends/circomcompat.nim

```diff
@@ -51,25 +83,18 @@ proc release*(self: CircomCompat) =
 
 proc prove*[H](
```

Comment:
should this be public?

Reply:
Guess it doesn't have to. Will make it private.

Reply:
Fixed!